### PR TITLE
Lps 90074 4

### PR DESCRIPTION
**Pull request by Preston-Crary**

### CODE REVIEW

File: modules/apps/headless/headless-workflow/headless-workflow-impl/src/main/java/com/liferay/headless/workflow/internal/mutation/v1_0/Mutation.java [hunk 1]
```diff
@@ -17,12 +17,6 @@
 import com.liferay.headless.workflow.dto.v1_0.WorkflowTask;
 import com.liferay.headless.workflow.resource.v1_0.WorkflowTaskResource;
 
-import graphql.annotations.annotationTypes.GraphQLField;
-import graphql.annotations.annotationTypes.GraphQLInvokeDetached;
-import graphql.annotations.annotationTypes.GraphQLName;
```
Comment (Preston-Crary) [inline modules/apps/headless/headless-workflow/headless-workflow-impl/src/main/java/com/liferay/headless/workflow/internal/mutation/v1_0/Mutation.java]: So just to write it down. Keep the graphql.annotations.annotationTypes usages but remove the DataFetchingEnvironment parameters.

Comment (Preston-Crary) [reply modules/apps/headless/headless-workflow/headless-workflow-impl/src/main/java/com/liferay/headless/workflow/internal/mutation/v1_0/Mutation.java]: So not what this commit is doing.

**Comment by Preston-Crary** [comment]
@petershin pushed f518fbf for what we really want to have generated.

**Comment by petershin** [comment]
Sent to Preston: https://github.com/Preston-Crary/liferay-portal/pull/682

**Pull request closed without merging**